### PR TITLE
hide practice levels for junior

### PR DESCRIPTION
**Pull request by smallst**
image
image
fix ENG-1061

### CODE REVIEW

File: ozaria/site/components/teacher-dashboard/BaseCurriculumGuide/components/ModuleContent.vue [hunk 1]
```diff
@@ -105,13 +111,18 @@ export default {
       event.stopPropagation()
       event.preventDefault()
       const level = this.getModuleInfo?.[this.moduleNum].find(l => l.slug === identifier)
-      if (hideCode) {
-        this.showCodeLevelSlugs = _.without(this.showCodeLevelSlugs, identifier)
-        return
+      const relatedLevels = this.relatedLevels(level)
+      console.log('relatedLevels', relatedLevels)
```
Comment (nwinter) [inline ozaria/site/components/teacher-dashboard/BaseCurriculumGuide/components/ModuleContent.vue:115]:
```suggestion
```

**Review by nwinter — APPROVED** [review]
Awesome, thanks. Can you also make it so that we append `" Practice levels: N"` to each level description, if N is greater than 0, when it's the Junior course?

File: ozaria/site/components/teacher-dashboard/BaseCurriculumGuide/components/ModuleContent.vue (hunk 1 above)
Comment (smallst) [reply ozaria/site/components/teacher-dashboard/BaseCurriculumGuide/components/ModuleContent.vue:115]: oh okay

**Comment by mrfinch** [comment]
If we click See Code/ Hide code on the practice levels multiple times, they just hide 😅 . Not a major bug but if it's an easy fix, can we just hide Show/Hide buttons from Practice level for now like for 1a, 1b, 1c? 
Screenshot 2024-08-27 at 4 02 36 PM

**Comment by smallst** [comment]
good catch. hide/see code need to be fixed

**Pull request merged**